libs/rxjs-web/src/lib/types/observer.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * A generic notification type for rxjs-web
+ */
+export interface ObserverNotification<T = any, O = any> {
+  /**
+   * An array of entries returned from the notification API
+   */
+  entries: ReadonlyArray<T>;
+  /**
+   * The Observer from the notification API
+   */
+  observer: Readonly<O>;
+}
+export type IntersectionNotification = ObserverNotification<
```

Comment:
from a file structuring point of view I'd prefer having the explicit types near by the operator, so IntersectionNotification should be in the same file as the creation operator